### PR TITLE
PEP-8 fixes for ServiceRouter

### PR DESCRIPTION
**Pull request by cmaloney**
Tested with python2 `pep8` command

- four space indents
- one newline empty line max in function bodies
- Wrap at 80 columns

### CODE REVIEW

**Comment by kolloch** [comment]
Hi @ssk2, can you review this? It's mostly whitespace changes but I barely know Python.

File: bin/servicerouter.py [hunk 1]
```diff
@@ -154,91 +154,91 @@ class ConfigTemplater(object):
 '''
 
     def __init__(self, directory='templates'):
-       self.__template_dicrectory = directory
-       self.__load_templates()
+        self.__template_dicrectory = directory
```
Comment (ssk2) [inline bin/servicerouter.py]: Typo here, can we fix this?

File: bin/servicerouter.py [hunk 2]
```diff
@@ -630,12 +646,16 @@ def get_arg_parser():
     parser.add_argument("--marathon", "-m",
                         required=True,
                         nargs="+",
-                        help="Marathon endpoint, eg. -m http://marathon1:8080 -m http://marathon2:8080")
+                        help="Marathon endpoint, eg. " +
+                             "-m http://marathon1:8080 " +
+                             "-m http://marathon2:8080")
     parser.add_argument("--listening", "-l",
-                        help="The HTTP address that marathon can call this script back at (http://lb1:8080)")
+                        help="The HTTP address that marathon can call this " +
```
Comment (ssk2) [inline bin/servicerouter.py]: `marathon` -> `Marathon`

File: bin/servicerouter.py [hunk 3]
```diff
@@ -630,12 +646,16 @@ def get_arg_parser():
...
+                        "script back at (http://lb1:8080)")
+    log_socket = "/var/run/syslog" if sys.platform == "darwin" else "/dev/log"
```
Comment (ssk2) [inline bin/servicerouter.py]: Can we give this a better name? Perhaps `default_log_socket`.

**Comment by ssk2** [comment]
@cmaloney - a few comments but otherwise fine.

**Comment by kolloch** [comment]
Hi @cmaloney , is this still going anywhere?

**Comment by cmaloney** [comment]
@kolloch: I was going to wait for #1444 to land then rebase this on top of it and fix the items pointed out by ssk2

**Comment by cmaloney** [comment]
@ssk2: Fixed up your issues in the second patch, can you double check that it looks good to you?

Re-ran `pep8` on the file to verify it's pep8 compliant with the rebasing (Cause of some of the additional changes).

**Comment by ssk2** [comment]
@cmaloney, looks good, thanks! I'll let you merge - looks like some build issues (may or may not be because of this..)

**Pull request merged**